docs/package.json

```diff
@@ -6,18 +6,18 @@
   "main": "index.js",
   "types": "index.d.ts",
   "scripts": {
-    "build": "npm run generate:schemas",
-    "test": "npm run build && npm run lint:json && npm run lint:yaml && npm run validate:schemas && npm run deploy:docs",
+    "build": "rimraf .tmp && run-s generate:schemas generate:resolved-spec generate:docs generate:postman",
+    "test": "rimraf .tmp && run-s lint:yaml lint:json lint:openapi validate:schemas",
     "lint:yaml": "yamllint ./openapi.yaml",
     "lint:json": "jsonlint-cli ./api/**/*.json ./entities/**/*.json",
     "lint:openapi": "lint-openapi ./openapi.yaml",
     "generate:types": "ts-node ./scripts/generate-types.ts",
     "generate:schemas": "gulp && npm run generate:types",
-    "generate:resolved-spec": "export API_VERSION=$(shx tail -n 1 ../.git-info) && speccy resolve --output .tmp/openapi.resolved.yaml openapi.yaml && shx sed -i 'STACKS_API_VERSION' ${API_VERSION:-1.0.0} .tmp/openapi.resolved.yaml > /dev/null",
+    "generate:resolved-spec": "npm run git-info --prefix .. && export API_VERSION=$(shx tail -n 1 ../.git-info) && speccy resolve --output .tmp/openapi.resolved.yaml openapi.yaml && shx sed -i 'STACKS_API_VERSION' ${API_VERSION:-1.0.0} .tmp/openapi.resolved.yaml > /dev/null",
```

Comment:
Can you take this opportunity to rename the git info rule to `generate:git-info` ?
I should've done this in the original PR

Reply:
Done!